gateway/assets/header.html

```diff
@@ -0,0 +1,12 @@
+<header id="header">
+  <div class="ipfs-logo">&nbsp;</div>
+  <nav>
+    <a href="https://ipfs.tech" target="_blank" rel="noopener noreferrer">About<span class="dn-mobile"> IPFS</span></a>
+    <a href="https://ipfs.tech#install" target="_blank" rel="noopener noreferrer">Install<span class="dn-mobile"> IPFS</span></a>
+    {{ with .SupportURL }}
+    <a href="{{ . }}" target="_blank" rel="noopener noreferrer" title="Support">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18.4 21"><circle cx="7.5" cy="4.8" r="1"/><circle cx="11.1" cy="4.8" r="1"/><path d="M12.7 8.4c-0.5-1.5-1.9-2.5-3.5-2.5 -1.6 0-3 1-3.5 2.5H12.7z"/><path d="M8.5 9.7H5c-0.5 0.8-0.7 1.7-0.7 2.7 0 2.6 1.8 4.8 4.2 5.2V9.7z"/><path d="M13.4 9.7H9.9v7.9c2.4-0.4 4.2-2.5 4.2-5.2C14.1 11.4 13.9 10.5 13.4 9.7z"/><circle cx="15.7" cy="12.9" r="1"/><circle cx="15.1" cy="15.4" r="1"/><circle cx="15.3" cy="10.4" r="1"/><circle cx="2.7" cy="12.9" r="1"/><circle cx="3.3" cy="15.4" r="1"/><circle cx="3.1" cy="10.4" r="1"/></svg>
```

Comment:
```suggestion
      Support
```

We may want to replace this 🐞 icon by a more generic message, such as "Support". Another possibility is to allow the implementer to add as many more menu buttons as they want. So each gateway using this library can add as many more things next to "About IPFS" and "Install IPFS" as they want. This would replace `SupportURL` by something like `[]MenuItem`. What do you think @lidel?

Reply:
Done in [ffe46ea](https://github.com/ipfs/boxo/pull/298/commits/ffe46ea5facad5751627b6b1c9f87288bcb9b3be).